integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/reactive/DefaultReactiveGraphResultSetIT.java

```diff
@@ -36,14 +35,12 @@
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
-import org.junit.experimental.categories.Category;
 import org.junit.rules.RuleChain;
 import org.junit.rules.TestRule;
 import org.junit.runner.RunWith;
 
 @DseRequirement(min = "6.8.0", description = "Graph paging requires DSE 6.8+")
 @RunWith(DataProviderRunner.class)
-@Category(ParallelizableTests.class)
 public class DefaultReactiveGraphResultSetIT {
```

Comment:
As mentioned earlier, I think it's better to keep this parallelizable and change the rule below.

Reply:
I didn't even think about trying that, but I will.

Reply:
Actually, I don't think I can switch this to use `CcmRule` as the Rule needs to be able to set the "graph" workload.

Reply:
Ouch, you are right. We should investigate if it's possible to add the graph workload to the default CCM cluster for parallelizable tests.

Reply:
Created JAVA-2723.